CHANGELOG.md

```diff
@@ -6,6 +6,45 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 (modification: no type change headlines) and this project adheres to
 [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [4.1.1] - 2019-11-19
+
+First stable `Istanbul` release passing all `StateTests` and `BlockchainTests`
+from the official Ethereum test suite
+[v7.0.0-beta.1](https://github.com/ethereum/tests/releases/tag/v7.0.0-beta.1).
+Test suite conformance have been reached along work on
+PR [#607](https://github.com/ethereumjs/ethereumjs-vm/pull/607) (thanks @s1na!)
+and there were several fixes along the way, so it is strongly recommended that
+you upgrade from the first `beta` `Istanbul` release `v4.1.0`.
+
+**Istanbul Related Fixes**
+
+- Refund counter has been moved from the `EEI` to the `EVM` module,
+  PR [#612](https://github.com/ethereumjs/ethereumjs-vm/pull/612), `gasRefund`
+  is re-added to the `execResult` in the `EVM` module at the end of message
+  execution in `EVM` to remain (for the most part) backwards-compatible in the
+  release
+- Fixed `blake2f` precompile for rounds > `0x4000000`
+- Fixed issues causing `RevertPrecompiled*` test failures
+- Fixed an issue where the `RIPEMD` precompile has to remain _touched_ even
+  when the call reverts and be considered for deletion,
+  see [EIP-716](https://github.com/ethereum/EIPs/issues/716) for context
+- Updated `ethereumjs-block` to `v2.2.1`
```

Comment:
Also updated `ethereumjs-blockchain` to `v4.0.2`

Reply:
Updated, ready for re-review.

Reply:
Would fit well if you could short-term reapprove, thanks!